### PR TITLE
Add docs for framework statistics

### PR DESCRIPTION
**Pull request by pmuens**
## What did you implement:

Add documentation for new framework statistics implementation.

## How did you implement it:

Added / updated docs / `.md` files.

## How can we verify it:

Just look through the `docs` folder.

## Todos:

- [x] Write documentation
- [x] Provide verification config/commands/resources
- [x] Change ready for review message below

***Is this ready for review?:*** YES

### CODE REVIEW

**Comment by pmuens** [comment]
@flomotlik Ready for review

File: docs/framework-statistics.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# Framework statistics
+
+Serverless will automatically collect anonymous framework statistics. This is done so that we better understand the usage and needs
+of our users to improve Serverless in future releases. However you can always [disable it](#how-to-disable-it).
+
+## What we collect
+
+Our main goal is anonymity. All the data is anonymized and won't reveal who you are or what the project you're working on is / looks like.
+
+Please take a look at the [`logStat()` method](../lib/classes/Utils.js) in the `Utils` class to see what (and how) we collect statistics.
```
Comment (mthenw) [inline docs/framework-statistics.md]: absolute URL is needed here. See https://github.com/serverless/serverless/pull/2369

File: docs/framework-statistics.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## How it's implemented
+
+We encourage you to look into the source to see more details about the actual implementation.
+
+The whole implementation consists of two parts:
+
+1. The [slstats plugin](../lib/plugins/slstats)
+2. The `logStat()` method you can find in the [Utils class](../lib/classes/Utils.js)
```
Comment (mthenw) [inline docs/framework-statistics.md]: absolute url

**Comment by flomotlik** [comment]
GTM after changes that @mthenw asked for are implemented.

**Comment by pmuens** [comment]
Thanks for the merge.

I would've not changed the links an keep them relative throughout the codebase.
@DavidWells is working on a fix in the docs build setup which will automatically convert the relative links to correct paths.

**Comment by flomotlik** [comment]
Can you send a pr for that relative link

Am 14.10.2016 17:17 schrieb "Philipp Muens" <notifications@github.com>:

**Comment by pmuens** [comment]
Yes, I opened up a separate issue so that we can keep track of it: https://github.com/serverless/serverless/issues/2380

**Pull request merged**